### DISCUSSION TITLE
how to send photo with file_id?

### DISCUSSION THREAD

**Discussion by FatihArridho** [telegraf/telegraf#1821] [Q&A]
anyone want to help me solve this problem?

ctx.replyWithPhoto('file_id') or ctx.sendPhoto('file_id')

but the result is like this:
Error: 400: Bad Request: can't use file of type ChatPhoto as Photo

**Comment by MKRhere** [comment]
You cannot use `ChatPhoto` directly with `file_id`. You'll have to download and reupload it. In fact, you cannot mix any two types of files by their IDs.

**Reply by FatihArridho** [reply]
> You cannot use `ChatPhoto` directly with `file_id`. You'll have to download and reupload it. In fact, you cannot mix any two types of files by their IDs.

can you help how to send a profile photo of a telegram user?
